webpack.config.js

```diff
@@ -0,0 +1,40 @@
+let path = require('path');
+let nodeExternals = require('webpack-node-externals');
+const moduleObj = {
+    loaders: [
```

Comment:
that's the old way of using webpack, check this for the rules set https://webpack.js.org/configuration/module/#rule-loaders